libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -171,13 +172,19 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
   }
 
   private observeScrollElementResize() {
-    if (!this.ionContent) return;
-
+    if (
+      !this.ionContent ||
+      !this.ionContent.getScrollElement ||
+      !this.ionContent.getScrollElement() ||
+      !this.ionContent.getScrollElement().then
```

Comment:
The API for ionContent always has `getScrollElement()` method that returns a promise => fix in spec instead...